setup.py

```diff
@@ -28,46 +31,34 @@
 Topic :: Software Development :: Libraries :: Python Modules"""
 
 
-def setup_cclib():
-
-    # Import from setuptools only if requested.
-    if 'egg' in sys.argv:
-        sys.argv.pop(sys.argv.index('egg'))
-        from setuptools import setup
+def readme():
```

Comment:
I don't see this function being called anywhere in the file or the rest of the codebase. Is it for PyPI? If so, can you add a comment? Otherwise, it seems unnecessary.

Reply:
Oh yes, it is unnecessary. Sorry.
In the projects I saw, it was always used like this:
```
long_description=readme()
```
you have the approach with the ``__doc__`` variable, which looks cleaner.